test/logging_test.dart

```diff
@@ -714,5 +714,15 @@ void main() {
       expect(records[1].error, isNotNull);
       expect(records[2].error, isNull);
     });
+
+    test('listen for level changed', () {
```

Comment:
We should add a test that setting it to the current level does _not_ produce an event, since we do have logic for that

Reply:
Right, I've just added the test.

Reply:
Right, I've just added the test.